### PR TITLE
Feature/component wireframe

### PR DESCRIPTION
**Pull request by KatherineBlaine**
### Notes: (includes questions, ideas for the future, anything you want to call out) 
Async work:
- KB: Activities component
- CC: Journal entry component
- AS: Feelings log component
- JF: Feelings component

### CODE REVIEW

**Comment by chrissycooper** [comment]
Look great, thanks for setting it up KB

**Review by jfogiato — APPROVED** [review]
Solid work - thanks for the initial buildout and driving for us, KB!!

File: src/Components/Activities/Activities.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+const Activities = () => {
+  return (
+    <>
+    <div>
```
Comment (chrissycooper) [inline src/Components/Activities/Activities.js:6]: indentation could be adjusted

**Review by chrissycooper — APPROVED** [review]

**Review by espressoGoddess — APPROVED** [review]
Looks good!

**Pull request merged**